packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -451,6 +451,12 @@ class _CupertinoTextFieldState extends State<CupertinoTextField> with AutomaticK
     _renderEditable.handleTapDown(details);
   }
 
+  void _handleForcePressStarted(ForcePressDetails details) {
+    _renderEditable.setLastTapDownPosition(details.globalPosition);
```

Comment:
I didn't read the rest yet but I don't think we need this.

Reply:
Add a selectWordAt API on the render editable (we can pre-empt the near future with a from and to argument)